README.md

```diff
@@ -89,22 +89,18 @@ rm -r app/demo && rm rm -r app/git ./gradlew :app:generateTestData
 
 ## Docker build
 The app is dockerised by running the `./buildkite/build-app.sh` script, which does the following:
-1. Calls `./buildkite/make-build-env.sh` which builds a docker image based on the `Dockerfile` which contains all the gradle and npm dependencies needed to 
-distribute the app. This image will also be re-used for the blackbox tests.
+1. Calls `./buildkite/make-build-env.sh` which builds a docker image based on the `Dockerfile` which contains all the
+gradle and npm dependencies needed to distribute the app. This image will also be re-used for the blackbox tests.
 1. Builds the app specific build environment image based on `app.Dockerfile` which inherits from the above.
 1. Generates an orderly-web database containing test data with `./buildkite/make-db.sh`
 1. Runs all dependencies needed for tests as a docker network
 1. Runs the image created in step 2. which tests the app and if successful, uploads coverage to Codecov and runs the
-`distDocker` task which builds and pushes the final docker image containing just the compiled app. Note that `CI_ENV` is
-used to pass relevant variables from the Buildkite environment into the Docker container for coverage upload,
-specifically metadata on the most recent git revision and the Codecov upload token.
+`distDocker` task which builds and pushes the final docker image containing just the compiled app.
```

Comment:
Is this now slightly inaccurate, because it's no longer the distDocker task which pushes the image - that's now done in a separate command, only if all the other tasks (test, linting etc) succeeded?

Reply:
Thanks - documentation amended. The behaviour previously was a bit odd - the `distDocker` task uploaded the Docker image with a commit tag. It was then pushed again using `docker push`. The branch tag was just pushed using `docker push`.